README.md

```diff
@@ -63,26 +80,32 @@ Download the docker-compose.yml, the .env and the setup.sh files:
 
 #### Create a docker-compose environment
 
-- Edit the `.env` to your liking. Adapt this file with your FQDN.
-  - This file supports only simple `VAR=VAL` lines (see [Documentation](https://docs.docker.com/compose/env-file/)).
-  - Don't quote your values.
-  - Variable substitution is *not* supported (e.g. `OVERRIDE_HOSTNAME=$HOSTNAME.$DOMAINNAME`).
+- The `.env` file contains the default environmental variables and their explanations. You can also find them in the more readable file [ENV.MD](ENV.MD). These variables will be passed to `docker-compose.yml` during running. You can change these variables in `docker-compose.yml`.
 - Install [docker-compose](https://docs.docker.com/compose/) in the version `1.7` or higher.
 
-#### Start Container
-    docker-compose up -d mail
-
-#### Create your mail accounts
+#### Create your mail accounts (at least one account)
```

Comment:
agree with leaving out the "at least one account"

Reply:
But this is what the mailserver tells me to do
`mail    | Error: Unless using LDAP, you need at least 1 email account to start the server.`
It keeps complaining unless I do this.

Reply:
Message comes from https://github.com/tomav/docker-mailserver/blob/9e7348c8f93de24d3d5f796f0b8a4ef09b9968d9/target/start-mailserver.sh#L626-L631

So it is only necessary to create an account if you are not using LDAP.